### PR TITLE
Update banner and compliance page copy

### PR DESCRIPTION
**Pull request by daniellemaxwell**
### Scope of changes

- Updates Envoy banner copy.
- Adds details about integrating TRISA with Envoy to the `Integrate & Comply` page.
- Checks for an error config before attempting to set the retry value.

Note: `.po` files and a snapshot have also been updated. These do not require a review as they are automatically generated.

[Comply page Figma design](https://www.figma.com/design/fHC69EVhfNtsKef1yKgW33/TRISA_GDS_VASP_UI?node-id=7096-87&t=rOXzAwmzbFCZPKjw-4)

### Type of change

- [x] bug fix
- [ ] new feature
- [ ] documentation
- [x] other (describe)

### Acceptance criteria

https://www.awesomescreenshot.com/video/28968107?key=40bfeefbc59f8d9919841e6957404e18

### Author checklist

- [x] I have manually tested the change and/or added automation in the form of unit tests or integration tests
- [ ]  I have updated the dependencies list
- [ ]  I have recompiled and included new protocol buffers to reflect changes I made
- [ ]  I have added new test fixtures as needed to support added tests
- [x]   Check this box if a reviewer can merge this pull request after approval (leave it unchecked if you want to do it yourself)
- [x]  I have moved the associated Shortcut story to "Ready for Review"

### Reviewer(s) checklist

- [ ] Any new user-facing content that has been added for this PR has been QA'ed to ensure correct grammar, spelling, and understandability.

### CODE REVIEW

File: web/gds-user-ui/src/components/Banner/LandingBanner.tsx [hunk 1]
```diff
@@ -14,8 +14,8 @@ const LandingBanner = () => {
       <AlertIcon/>
       <AlertDescription fontWeight={'semibold'}>
         <Link href="https://calendar.app.google/FBg7GTmgDfeMbUMT9" isExternal>
-          <Trans id="Schedule a demo to learn about TRISA's open source self-hosted solution for cost-effective Travel Rule compliance.">
```
Comment (daniellemaxwell) [inline web/gds-user-ui/src/components/Banner/LandingBanner.tsx:17]: The id isn't required to extract the text to a `.po` file for translation, so it has been remove here and in the other component file.

File: web/gds-user-ui/src/components/Section/IntegrateAndComply.tsx [hunk 2]
```diff
@@ -61,124 +60,168 @@ export default function IntegrateAndComply() {
...
-                    can help your VASP set up your TRISA server and maintain it.">
-                      Talk to a member of your technical team to determine the requirements and
-                      resources to integrate TRISA with your system. Have members of your technical
-                      team integrate your systems with TRISA. Or work with a solutions provider that
-                      can help your VASP set up your TRISA server and maintain it.
-                    </Trans>
-                  </Line>
-                  <Line title={t`3rd Party Travel Rule Providers`}>
-                    <UnorderedList color={'#1F4CED'}>
+              <Box pt={4}>
```
Comment (daniellemaxwell) [inline web/gds-user-ui/src/components/Section/IntegrateAndComply.tsx:84]: This differs from the Figma design as the table component extended outside the container even with a max width set. I attempted a grid with 3 columns, but then ran into an issue with some of the text inside a grid item aligning in the center although a text align style had not been applied.

File: web/gds-user-ui/src/utils/axios.ts [hunk 3]
```diff
@@ -47,10 +47,10 @@ axiosInstance.interceptors.response.use(
     return response;
   },
   async (error: any) => {
-    // let _retry = 0;
     const originalRequest = error?.config;
-    originalRequest._retry = originalRequest?._retry || 0;
-    //
+    if (originalRequest) {
```
Comment (daniellemaxwell) [inline web/gds-user-ui/src/utils/axios.ts:51]: This will hopefully resolve an undefined error since adding optional chaining `(?.)` didn't work.

**Review by pdamodaran — APPROVED** [review]
Looks good @daniellemaxwell!

**Pull request merged**